Add release note for 1.21.0 and 1.16.1 (#4581)
line/armeria: site/src/pages/release-notes/1.16.1.mdx

---
date: 2022-12-06
---

## 🛠️ Bug fixes

> - You can now use the exception raised in a decorator to make a decision for retrying. #4558 #4559

Reviewer: ```suggestion
- Exceptions raised in decorators can now be used to determine whether to retry in <type://RetryingClient>. #4558 #4559
```

> - You can now use the exception, which is raised in the decorator, to make a decision for retrying. #4558 #4559

Reviewer: I think this can be simplified

```suggestion
- You can now use the exception raised in a decorator to make a decision for retrying. #4558 #4559
```

## 🙇 Thank you

<ThankYou usernames={[
  'ikhoon',
  'jrhee17',
  'Lincong',

>   'minwoox',

Reviewer 2: ```suggestion
  'minwoox'
```

Author: 😱
